src/coreclr/ToolBox/superpmi/superpmi-shared/spmirecordhelper.h

```diff
@@ -193,52 +223,166 @@ inline MethodContext::Agnostic_CORINFO_SIG_INFO SpmiRecordsHelper::CreateAgnosti
     return sig;
 }
 
+inline void SpmiRecordsHelper::StoreAgnostic_CORINFO_SIG_INST_HandleArray(
+    unsigned handleInstCount,
+    CORINFO_CLASS_HANDLE* handleInstArray,
+    DenseLightWeightMap<DWORDLONG>*& handleMap, // If we initialize it, the pointer gets updated.
+    /* OUT */ DWORD* handleInstCountOut,
+    /* OUT */ DWORD* handleInstIndexOut)
+{
+    unsigned handleInstIndex;
```

Comment:
maybe just set this to -1 and remove the else below?

Reply:
Certainly do-able. I'm kind of a fan of "aggressively simple and explicit logic" (my term), but it's just a style thing